rio_tiler/colormap.py

```diff
@@ -169,7 +169,7 @@ def register(self, name: str, custom_cmap: Union[Dict, str], force: bool = False
 
         """
         if not force and name in self._data.keys():
-            raise Exception(
+            raise AlreadyRegistered(
```

Comment:
Maybe a little more verbose?
```suggestion
            raise CmapAlreadyRegistered(
```

Reply:
I'd personally suggest `ColormapAlreadyRegistered` instead of `cmap`